frontend/controller/actions/identity.js

```diff
@@ -1,14 +1,23 @@
 'use strict'
 
 import sbp from '@sbp/sbp'
+import { keyId, keygen, deriveKeyFromPassword, serializeKey, encrypt } from '@utils/crypto.js'
```

Comment:
This file really should be part of Chelonia, could you move it there?

Reply:
Sure. Although it could also be more of a generic crypto library that Chelonia uses (will there be uses of `crypto.js` somewhere else?)

Reply:
Well, Chelonia will use it, and the contracts we create will use it. Also the backend will use it by-proxy of Chelonia (to verify signatures of messages it receives)